### ISSUE TITLE
How Do I print to see all the layers added to the optimizer?

### ISSUE THREAD

**Issue by gjhhust** [open-mmlab/mmdetection#10579]
This is the Swin configuration for its different layers to modify the learning rate：
````
paramwise_cfg=dict(
        custom_keys={
            'absolute_pos_embed': dict(decay_mult=0.),
            'relative_position_bias_table': dict(decay_mult=0.),
            'norm': dict(decay_mult=0.)
        })
````
I tried to add the [ Topformer ](https://github.com/hustvl/TopFormer ) network as backbone to mmdetection, but the training was very poor. I guess there were parameters that weren't added to the optimizer. I wanted to print and view all the layers in the optimizer, i wish to rule this out

Second, I have doubts about custom. Should the naming and code variables in custom be the same? Forgive me for asking, but I think this is a common problem. Topformer is the backbone of the TopFormer implemented in MMSEG, which has [ line 124 of the configuration file ](https://github.com/hustvl/TopFormer/blob/main/local_configs/topformer/topformer_base_1024x512_80k_2x8city.py), custom keys is: { Pos, head, Norm } , but I don't understand which layers these three keys correspond to, and I think it should be where there are custom parameter sets in the code, rather than where the names are directly related
